umap/static/umap/js/umap.js

```diff
@@ -352,6 +267,12 @@ U.Map = L.Map.extend({
     }
   },
 
+  overrideSchema: function (schema) {
+   for (const [key, extra] of Object.entries(schema)) {
+     U.SCHEMA[key] = L.extend({}, U.SCHEMA[key], extra)
```

Comment:
If we modify the object like this, why not using a class+object instead of a global? (or any other solution, but uppercase makes me think it's a constant, which it isn't)

Reply:
I'd say it is a constant, like Django settings are. It's a constant for a given uMap instance, but can be overrided just like we override django settings, once and for all.

Note that this generic overriding of the schema will be removed, as per discussion with @davidbgk . We'll keep it in mind but prefer to introduce it yet: this would give people managing a uMap instance a lot of new possibilities (like overriding all the default, eg. the default color, or weight, etc., or changing a help text, a placeholder, etc.), but for this exact reason we think we are not ready yet to assume those new possibilities and to support them on the long run. Needs more thinking.

Reply:
I really like the use case 👍 

My remark is more on the naming and semantics of it. If it is overridden by our own code, we might not want to use an uppercase constant, as (to me, at least) it conveys the meaning that it won't be changed, which isn't the case.

I won't die on this hill, though: it's mainly cosmetic 💅